internal/errors/errors.go

```diff
@@ -174,4 +174,9 @@ var (
 
 	// As represents errors.As.
 	As = errors.As
+
+	// errNilCustomError represents a function to generate an error that given name's error object is nil.
+	errNilCustomError = func(name string) string {
```

Comment:
Should we define pure function instead of function variable?:thinking:

Reply:
Up to us.

Reply:
got it.